homework-16_DataManipulation.sql

```diff
@@ -0,0 +1,31 @@
+DELETE FROM "RESERVATIONS";
+DELETE FROM "ROOMS";
+DELETE FROM "GUESTS";
+DELETE FROM "HOSTS";
+DELETE FROM "USERS";
+
+INSERT INTO "USERS" (id, name) VALUES (1, 'Host and Guest 1');
+INSERT INTO "USERS" (id, name) VALUES (2, 'Guest 2');
+INSERT INTO "USERS" (id, name) VALUES (3, 'Host and Guest 3');
+
+INSERT INTO "HOSTS" (id) VALUES (1);
+INSERT INTO "HOSTS" (id) VALUES (3);
+
+INSERT INTO "GUESTS" (id) VALUES (1);
+INSERT INTO "GUESTS" (id) VALUES (2);
+INSERT INTO "GUESTS" (id) VALUES (3);
+
+INSERT INTO "ROOMS" (id, host_id, name, residents, price, aircondition, refrigerator) VALUES (1, 1, 'Room 1', 1, 300, true, true);
+INSERT INTO "ROOMS" (id, host_id, name, residents, price, aircondition, refrigerator) VALUES (2, 1, 'Room 2', 1, 100, false, false);
+INSERT INTO "ROOMS" (id, host_id, name, residents, price, aircondition, refrigerator) VALUES (3, 3, 'Room 3', 1, 200, false, true);
+
+INSERT INTO "RESERVATIONS" (guest_id, room_id) VALUES (1, 3);
+INSERT INTO "RESERVATIONS" (guest_id, room_id) VALUES (2, 1);
+INSERT INTO "RESERVATIONS" (guest_id, room_id) VALUES (2, 2);
+
+SELECT u.id AS user_id, u.name AS user_name
```

Comment:
The query returns any user with the biggest amount of reservations. Could you please add the query to return all users with the biggest amount of reservations?

Reply:
Sorry, but it is out of scope. The request was about one user :)